app/views/home/download.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="heading-large">Preview and download your code</h1>
+<p>Your picker and dataset is available as two files:</p>
+<ul class="list">
+  <li>picker.html</li>
+  <li>data.json</li>
+</ul>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <details class="details--with-border">
```

Comment:
But surely there must be some reason it's used.  Like you could say a colour is generic but usually it's the meaning that's written.  Like something-alert instead of something-red.